src/main/modules/nunjucks/media-account-rejection-reasons-lookup.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
In /src/main/resources, there is a lookup file called "listLookup.json". Feel like this lookup file should be in the same place as the other lookup file as it is called by the controller not just the nunjucks filter.

Reply:
Moved this too :)